third-party/easyloggingpp/src/easylogging++.cc

```diff
@@ -2094,8 +2108,10 @@ Storage::Storage(const LogBuilderPtr& defaultLogBuilder) :
   addFlag(LoggingFlag::AllowVerboseIfModuleNotSpecified);
 #if ELPP_ASYNC_LOGGING
   installLogDispatchCallback<base::AsyncLogDispatchCallback>(std::string("AsyncLogDispatchCallback"));
+  ELPP_INTERNAL_INFO(1, "ELPP ASYNC logger");
 #else
   installLogDispatchCallback<base::DefaultLogDispatchCallback>(std::string("DefaultLogDispatchCallback"));
+  ELPP_INTERNAL_INFO(1, "ELPP sync logger");
```

Comment:
"ELPP Sync logger selected", also adjust the above line accordingly

Reply:
done